tests/post/features/salt_api.feature

```diff
@@ -1,14 +1,5 @@
 @post @ci @local @salt
 Feature: SaltAPI
-    Scenario: Login to SaltAPI using Basic auth
-        Given the Kubernetes API is available
-        When we login to SaltAPI as 'admin' using password 'admin'
-        Then we can ping all minions
-        And we can invoke '[".*"]' on '*'
-        And we have '@wheel' perms
-        And we have '@runner' perms
-        And we have '@jobs' perms
-
```

Comment:
Maybe we should have a check that `Basic auth` not longer work (to test for example that after an upgrade we can't use Basic auth)

Reply:
I am going  to add